braze/client.py

```diff
@@ -28,15 +28,15 @@ class AppboyClient(object):
         print r['client_error']
         print r['errors']
     """
-    API_URL = 'https://api.appboy.com'
-
+    DEFAULT_API_URL = 'https://api.appboy.com'
     USER_TRACK_ENDPOINT = '/users/track'
     USER_DELETE_ENDPOINT = '/users/delete'
 
     REQUEST_POST = 'post'
 
-    def __init__(self, api_key):
+    def __init__(self, api_key, api_url=None):
         self.api_key = api_key
+        self.api_url = api_url or self.DEFAULT_API_URL
```

Comment:
Braze may give different customers (such as enterprise customers) a different base endpoint. This change allows the api_url to be specified when instantiating a client instance. If missing it defaults to `https://api.appboy.com`.